<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:1px solid white;
border-radius:10px;
spacing:10px;
padding:5px 5px;</string>
   </property>
   <property name="text">
    <string>       Retour</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSeekBackward"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="SoundCheckBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>138</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
margin-left:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="soundLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;8514oem&quot;;</string>
   </property>
   <property name="text">
    <string>Sound:</string>
   </property>
  </widget>
  <widget class="QLabel" name="musicLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;8514oem&quot;;</string>
   </property>
   <property name="text">
    <string>Music:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MusicCheckBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
margin-left:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="LangBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:blue;
max-width:150px;
padding: 5px 3px;
margin-right:100px;
</string>
   </property>
   <item>
    <property name="text">
     <string>Anglais</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Francais</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arabe</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LangLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;8514oem&quot;;</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
